tfx/tools/docker/requirements.txt

```diff
@@ -0,0 +1,356 @@
+# The following versions are manually captured to avoid unfinished dependency resolution.
```

Comment:
It might be good to note here that this file is here because our project dependencies are complex, and without using these constraints, pip fails to solve when installing the project; otherwise new contributors may think these are actual project dependencies.

Reply:
Thanks! I've added comments.